features/comments_and_kudos/comments_delete.feature

```diff
@@ -47,4 +47,23 @@ Feature: Delete a comment
     Then I should see "Comment deleted."
       And I should see "(Previous comment deleted.)"
       And I should see "I didn't mean that"
-  
+
+  Scenario: Deleting higher-level comments in a deep comment thread should still allow readers to access the deeper comments.
+
+    Given I am logged in as "author"
```

Comment:
If the name of the work creator's not important -- it doesn't seem to be -- I think you could do just:

```
Given the work "Testing"
And I am logged in as "commenter"
```